datadog_lambda/wrapper.py

```diff
@@ -45,6 +45,10 @@
     extract_http_status_code_tag,
 )
 
+# Patch third-party libraries for tracing, must be done before importing any
+# handler code.
+patch_all()
```

Comment:
We are planning to deprecate `ddtrace.patch_all(...)` and instead encourage folks to use `import ddtrace.auto` (which executes a similar set of operations as `ddtrace-run`).

I don't think we can make this change in this PR due to the overhead of loading all ddtrace products and entrypoints but it's something we should investigate

Reply:
Thanks Munir. It's definitely something on our radar. Last I benchmarked the difference between `patch_all` and `ddtrace.auto`, it was significant.